pick

```diff
@@ -206,4 +216,7 @@ if __name__=='__main__':
   if args.write:
     pick.write_password()
     exit()
-  
+
+  if args.delete:
+    pick.delete_password()
```

Comment:
Once you change the `-d` argument to look for `nargs=1`, then `args.delete` will be a List containing the single argument:
```py
print args.delete
# ["somealias"]
```
With that you could change your call site to pass in the alias:
```py
if args.delete:
  pick.delete_password(args.delete[0])
```